### ISSUE TITLE
info/boxes: include computed page rotation

### ISSUE THREAD

**Issue by maxiride** [pdfcpu/pdfcpu#350]
A pdf file has multiple pages with a known format (A4 for instance, 210 x 297 mm), some of the pages however are in horizontal (+90°).

The below PDF has page 1 in landscape and page 2 portrait, however the width and height reported by the info command are the same.

I expected it to be:
- page 1: w=297.04 h=209.90
- page 2: w=209.90 h=297.04

```shell
federico.d@PC-FEDED MINGW64 ~/Projects/playground
$ pdfcpu.exe info -p 1,2 -u mm 3027495.pdf
pages: 1,2
         PDF version: 1.7
          Page count: 6
Page 1:
  MediaBox (mm) (0.00, 0.00, 209.90, 297.04) w=209.90 h=297.04 ar=0.71  = CropBox, TrimBox, BleedBox, ArtBox
Page 2:
  MediaBox (mm) (0.00, 0.00, 209.90, 297.04) w=209.90 h=297.04 ar=0.71  = CropBox, TrimBox, BleedBox, ArtBox
............................................
               Title:
              Author:
             Subject:
        PDF Producer: pdfcpu v0.3.11 dev
     Content creator: JasperReports Library version 6.1.0
       Creation date: D:20210701083436+02'00'
   Modification date: D:20210701083436+02'00'
............................................
              Tagged: No
              Hybrid: No
          Linearized: No
  Using XRef streams: Yes
Using object streams: Yes
         Watermarked: Yes
............................................
           Encrypted: No

```

Is this an expected behaviour? If so is there a different way to determine if a page is in horizontal?
The goal here is to detect pages orientation to rotate them to make a PDF file with all vertical pages, I thought that a very simple approach would have been to check width and height and if width > height a rotation needs to occur, sign of the rotation +/-90 is not important.

Slightly related to #349

**Comment by hhrutter** [comment]
Generally `pdfcpu info` reports the detected page media boxes which are independent of any rotation in effect.
This even more applies when you supply `-pages` which reports the PDF page boundaries. 
At the heart of this decision lies that pages are nodes in a page tree and the rotation is an inherited attribute.
What we could do as an enhancement is to show the effective(computed) page rotation.

See also corresponding comments at #349

**Comment by maxiride** [comment]
I understand the reasoning indeed.

> What we could do as an enhancement is to show the effective(computed) page rotation.

This, paired with the page's size could effectively help us differentiate two landscape\portrait scenarios, below a landscape example:

## Case A:

Page size reported as 210 x 297 (portrait) but rotated +90 degrees -> viewed\printed as landscape.

## Case B:

Page size reported as 297 x 210 (landscape) but rotated 0 degrees -> viewed\printed as landscape.

Both cases would need a -90° rotation to be transformed to portrait, however with the the angle alone or the page size alone in each case we wouldn't be able to tell if and by which amount to rotate the a page. The two information together are (I suppose) the minimum viable to determine an action to be taken.

**Comment by hhrutter** [comment]
Fixed with v0.3.12
